applications/red-hat-marketplace.adoc

```diff
@@ -0,0 +1,10 @@
+[id="red-hat-marketplace"]
+= The Red Hat Marketplace
+include::modules/common-attributes.adoc[]
+:context: red-hat-marketplace
+
+toc::[]
+
+The link:https://marketplace.redhat.com[Red Hat® Marketplace] is an open cloud marketplace that makes it easy to discover and access certified software for container-based environments that run on public clouds and on-premesis.
```

Comment:
Do we need the r-ball after Red Hat here? I don't think we do that anywhere else that I'm aware of

Reply:
whoops that might have snuck in during a copy/paste from the Marketplace team